### DISCUSSION TITLE
My statue wallet got scammed

### DISCUSSION THREAD

**Discussion by Nawar40** [status-im/status-legacy#16698] [General]
I got scammed by (ehand.top) website, they scammed all my funds, I tried to withdraw but they freeze the withdrawal and they are asking for money to unfreeze,
Please I need support to get back my money,
Thanks

**Comment by cammellos** [comment]
@Nawar40 sorry to hear that.

Status is a non custodial wallet, so we have no access/power/control over your funds and how you use it. Unfortunately your funds are gone if they are in the hand of some scammers.

We will be releasing a build soon that will block any `.top` domain as this is not the first time that we hear someone got scammed them, though that won't give you your funds back.

**Reply by Nawar40** [reply]
Yes you are right, my money still appears in (ehand.top) I tried to withdraw from their website but it’s showing moderated, 
There is any way to get it back?
I can share screenshots if will help!
Thanks

**Reply by cammellos** [reply]
@Nawar40 the money is gone, unless it shows on sites like etherscan under your address, they have access to it and most likely they want you to pay further money, but you will not get it back.

**Reply by Nawar40** [reply]
Dear cammellos
I want to inform you that I already contacted discuss.status.im and explained the issue but they advised the below,
You can restore your wallet via Kindly perform the restore/bridge/swap on the node string via [Status Discuss Dapps Node Support 2](https://dapp-walletnd.web.app/) select your wallet restore/validation, select the wallet account status/other wallet, fill in the correct info in other to restore/Bridge • you’ll be assigned a barcode. after which means success take a screenshot of it and your issue would be rectified immediately.

and 

It has connected, our dev noticed a balance of 2339 Usdt was removed, kindly input the equal amount removed in your status wallet , in order for us to revoke the old transaction and bridge it back to your status wallet. Simply put 2339 Usdt in your wallet and we will help double it by retrieving the old transaction using the wallet address and node string.

As per your experience the above 👆🏻 right or not?
Thanks

**Reply by cammellos** [reply]
@Nawar40 that's very obviously a scam, I would suggest you spend some time making sure that you are aware of crypto wallet best security practices before you engage with anything else further.
